### ISSUE TITLE
Nucleated Erythrocytes

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5955]
**Preferred term label:** Nucleated Erythrocytes

**Synonyms** Nucleated red blood cells

**Definition (free text, please give PubMed ID)** Presence of red blood cells with a nucleus., PMID:28633658

**Parent term (use hpo.jax.org/app)** Abnormal erythrocyte morphology HP:0001877

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding term.
